### ISSUE TITLE
RHEL9 resource-agents no longer contain cloud agents

### ISSUE THREAD

**Issue by marcelmamula** [linux-system-roles/ha_cluster#185]
`This is of course cloud specific so I am not sure if you want to implement it here as I dont see any cloud platform specific code`

RedHat_9 var file should be enhanced with __ha_cluster_fullstack_node_packages from main RedHat var file to cover for newly split package on RHEL9.
Package to add: resource-agents-cloud

It contains:
```
/usr/lib/ocf/resource.d/heartbeat/aliyun-vpc-move-ip
/usr/lib/ocf/resource.d/heartbeat/aws-vpc-move-ip
/usr/lib/ocf/resource.d/heartbeat/aws-vpc-route53
/usr/lib/ocf/resource.d/heartbeat/awseip
/usr/lib/ocf/resource.d/heartbeat/awsvip
/usr/lib/ocf/resource.d/heartbeat/azure-events
/usr/lib/ocf/resource.d/heartbeat/azure-events-az
/usr/lib/ocf/resource.d/heartbeat/azure-lb
/usr/lib/ocf/resource.d/heartbeat/gcp-ilb
/usr/lib/ocf/resource.d/heartbeat/gcp-pd-move
/usr/lib/ocf/resource.d/heartbeat/gcp-vpc-move-route
/usr/lib/ocf/resource.d/heartbeat/gcp-vpc-move-vip
```

Explanation:
RHEL9 changed resource-agents package and it no longer contains cloud resource agents. They are now under resource-agents-cloud.

Example for REL9.2 on AWS:
**resource-agents-4.10.0-34.el9_2.2.x86_64**
```
[root@rhel9ha0 ~]# rpm -ql resource-agents | grep aws-vpc-move-ip
/usr/share/man/man7/ocf_heartbeat_aws-vpc-move-ip.7.gz
[root@rhel9ha0 ~]#
```

**resource-agents-cloud-4.10.0-34.el9_2.2.x86_64**
```
[root@rhel9ha0 ~]# rpm -ql resource-agents-cloud | grep aws-vpc-move-ip
/usr/lib/ocf/resource.d/heartbeat/aws-vpc-move-ip
/usr/share/man/man7/ocf_heartbeat_aws-vpc-move-ip.7.gz
```

**Comment by tomjelinek** [comment]
Hi @marcelmamula,

Thank you for pointing this out. The fact that cloud resource and fence agents are not installed by default on RHEL is intentional and we prefer to keep it that way. The reason is that dependencies of those agents are rather large. This is why the agents were moved to separate packages in RHEL.

If those agents are needed, they can be easily installed by specifying appropriate packages in `ha_cluster_extra_packages` variable.

Note that there is no regression or change of behavior, as cloud agents are packaged separately since RHEL 8, and the role is compatible with RHEL 8.3 and newer.

**Comment by marcelmamula** [comment]
Thank you for clarification @tomjelinek 
I have used default values when doing testing and it was not apparent that it is needed. Maybe readme update to cover this requirement would help to avoid errors for users, if there is no plan to add it into defaults.

**Comment by tomjelinek** [comment]
This is being addressed in #194
